packages/jupyter-chat/src/widgets/chat-widget.tsx

```diff
@@ -321,6 +328,41 @@ export class ChatWidget extends ReactWidget {
     }
   }
 
+  /**
+   * Process dropped tabBar files
+   */
+  private _processTabDrop(event: Drag.Event) {
+    const factory = event.mimeData.getData(TABBAR_FILE_MIME) as () => any;
```

Comment:
```suggestion
    const factory = event.mimeData.getData(TABBAR_FILE_MIME) as () => Widget;
```

The factory should always return return a `Widget`.

Reply:
Done 🎉